ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh

```diff
@@ -56,8 +56,8 @@ GITHUB__APP__CLIENT_SECRET=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-c
 GITHUB__APP__WEBHOOK_SECRET=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-webhook-secret)
 GITHUB__APP__WEBHOOK_URL=GITHUB_APP_WEBHOOK_URL
 GITHUB__APP__PRIVATE_KEY=$(base64 -d < /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-private-key)
-GITOPS__GIT_TOKEN="dummy-token"
-QUAY__TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/quay-robot-password)
+GITOPS__GIT_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gihtub_token)
+QUAY__DOCKERCONFIGJSON=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhtap_quay_ci_token)
```

Comment:
We don't need QUAY__TOKEN anymore?

Reply:
`You need to replace QUAY__TOKEN with QUAY__DOCKERCONFIGJSON`, see the comment https://github.com/redhat-appstudio/rhtap-installer/pull/38#issuecomment-1986582015

Reply:
https://github.com/redhat-appstudio/rhtap-installer/pull/38#issuecomment-1986582015